src/common/low_precision_transformations/include/low_precision/broadcast.hpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "transparent_base_transformation.hpp"
+
+namespace ov {
+namespace pass {
+namespace low_precision {
+
+/**
+ * @ingroup ov_transformation_common_api
+ * @brief BroadcastTransformation propagates dequantization operations through Broadcast operation.
+ *
+ * For more details about the transformation, refer to
+ * [BroadcastTransformation](@ref openvino_docs_OV_UG_lpt_BroadcastTransformation) page
+ * in the OpenVINO Developer Guide.
+ */
+class LP_TRANSFORMATIONS_API BroadcastTransformation : public TransparentBaseTransformation {
```

Comment:
It would be great to cover the transformation with tests, at least functional

Reply:
has been started, was completed